frontend/src/pages/Login.jsx

```
@@ -0,0 +1,137 @@
+
+        {/* Link to Registration Page */}
+        <p className="text-center mt-4 text-sm text-gray-600">
+          Don't have an account?{' '}
+          <Link to="/registration" className="text-blue-500 hover:underline">
+            Register
+          </Link>
+        </p>
+
+        {/* Social Login Buttons */}
+        <div className="mt-6 space-y-3">
+          <button className="w-full bg-white border border-gray-300 p-3 rounded-lg flex items-center justify-center hover:bg-gray-50 transition duration-300">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)